viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from enum import unique
+
+
 def create_movie(title, genre, rating):
-    pass
+    dict = {}
+    if title and genre and rating:
+        dict['title'] = title
+        dict['genre'] = genre
+        dict['rating'] = rating
+        return dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watchlist = user_data['watchlist']
+    for watchlist_movie in watchlist:
+        if title == watchlist_movie['title']:
+            movie = watchlist_movie
+            result = add_to_watched(user_data, movie)
+            user_data['watchlist'].remove(watchlist_movie)
+            return result

```

Comment:
nice job reusing functions! We don't need to assign the return statement here. Because dictionaries are passed by reference, the resulting changes from `add_to_watched` have been made to `user_data`. We could do something like this:

```python
add_to_watched(user_data, movie)
user_data['watchlist'].remove(watchlist_movie)
```
and let the outer return statement do its job!